components/ProjectDetail/ProjectDetail.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+const ProjectContainer = styled.div`
+  background-color: #faf8f7;
+  margin-left: 20px;
+  padding: 0 10px 10px 10px;
+  margin-bottom: 20px;
+  border-radius: 5%;
+`;
+
+const StyledImage = styled(Image)`
+  height: 100%;
+  width: 100%;
+  border-radius: 5%;
+`;
+
+export default function ProjectDetail({
+  project: {
+    category,
+    title,
+    imageSource,
+    width,
+    height,
+    longDescription,
+    organizer,
+    contact,
+  },
+}) {
```

Comment:
Distructure you prop in the function body before your return statement:
```suggestion
}) {
const {category, title, mageSource, width, height, longDescription, organizer, contact} = project
```

Reply:
This did not work unfortunately. I got an error project not defined afterwards. Also tried that already yesterday.

Reply:
Strange, it worked for me.